### PR TITLE
Kaitie/fix safari styles on rubrics

### PR DESCRIPTION
**Pull request by kobryan0619**
During our Bug Bash, we noticed that the CSS in Safari was rendering differently than in Chrome.  This PR fixes those changes. 

image

Note: I checked with Molly about the bolded text and she confirmed that the text for the learning goal name should be bolded.

## Links

ticket

## Testing story

- I modified tests here and deleted one test since the arrows are now represented in the CSS instead of .jsx.  I know we don't typically test CSS in this type of test, so wasn't sure if that was needed here.

### CODE REVIEW

File: apps/test/unit/templates/rubrics/LearningGoalTest.jsx [hunk 1]
```diff
@@ -146,17 +146,6 @@ describe('LearningGoal', () => {
     expect(wrapper.find('AiToken')).to.have.lengthOf(0);
   });
 
-  it('shows down arrow when closed and up arrow when open', () => {
```
Comment (kobryan0619) [inline apps/test/unit/templates/rubrics/LearningGoalTest.jsx:149]: Since this was moved into the CSS, I didn't think this test was needed any more.

**Review by bethanyaconnor — APPROVED** [review]
Changes look good to me. It looks like the name of the learning goal got more bolded -- do you have any idea why?

File: apps/src/templates/rubrics/LearningGoal.jsx [hunk 2]
```diff
@@ -200,22 +201,8 @@ export default function LearningGoal({
     <details className={style.learningGoalRow}>
       <summary className={style.learningGoalHeader} onClick={handleClick}>
         <div className={style.learningGoalHeaderLeftSide}>
-          {isOpen && (
-            <FontAwesome
-              icon="angle-up"
-              onClick={() => setIsOpen(false)}
```
Comment (kobryan0619) [inline apps/src/templates/rubrics/LearningGoal.jsx:206]: @bethanyaconnor I just want to confirm that this is OK.  Given that we moved this to the CSS, this is going to now be tracked in the clicking of the `summary` element above instead.

Comment (bethanyaconnor) [reply apps/src/templates/rubrics/LearningGoal.jsx:206]: yep sounds fine to me!

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**